buildTests/AMDetectorSetTest/AMDetectorSetTest.pro

```diff
@@ -2,5 +2,5 @@ include ( $$PATH_TO_AM/compositeCommon/AMDetectorSet.pri )
 
 TARGET = AMDetectorSetTest
 
-SOURCES += \
+SOURCES *= \
```

Comment:
+= can be read as "always add" and *= can be read as "add if you haven't yet".  *= is preferred because if you doubly add a file into source and headers and they are not properly header scoped (#define CLASS_NAME_H) then you will get linker errors.